src/garage/tf/regressors/base.py

```diff
@@ -1,33 +1,37 @@
-"""Regressor base classes with Parameterized."""
+"""Regressor base classes without Parameterized."""
 import abc
 
-from garage.tf.core import Parameterized
+import tensorflow as tf
 
+from garage.misc.tensor_utils import flatten_tensors
+from garage.misc.tensor_utils import unflatten_tensors
 
-class Regressor(Parameterized, abc.ABC):
+
+class Regressor(abc.ABC):
     """
-    Regressor base class.
+    Regressor base class without Parameterized.
 
     Args:
-        input_shape: Input shape.
-        output_dim: Output dimension.
-        name: Name of the regressor.
+        input_shape (tuple): Input shape.
```

Comment:
when using containers, please specify the type of the contained objects (i.e. `tuple[int]`)